docs/ocis/deployment/continuous_deployment.md

```diff
@@ -11,67 +11,50 @@ geekdocFilePath: continuous_deployment.md
 
 We are continuously deploying the following deployment examples. Every example is deployed in two flavors:
 
-- Latest: reflects the current master branch state of oCIS and will be updated with every commit to master
-- Daily: reflects the master branch state of oCIS as of the time of deployment
-- Released: reflects the newest release state (currently latest release of version 1) and will be updated with every release
+- Master: reflects the current master branch state of oCIS and will be updated with every commit to master
+- Rolling: reflects the latest rolling release of oCIS and will be updated with every release
+- Production: reflects the latest production release of oCIS and will be updated with every release
 
 The configuration for the continuous deployment can be found in the [oCIS repository](https://github.com/owncloud/ocis/tree/master/deployments/continuous-deployment-config).
 
-# oCIS with Traefik
+# oCIS with Web Office
+
+This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with Collabora Online as the office suite, traefik reverse proxy, cloudimporter and the inbucket mail catching server to showcase the full feature set of oCIS.
```

Comment:
```suggestion
This deployment is based on our modular [ocis_full Example](ocis_full.md) and uses the default configuration with traefik reverse proxy, Collabora Online as the office suite, Tika for search and Cloudimporter to showcase the full feature set of oCIS. The Inbucket mail catching server or other services can be easily configured additionally.
```
I have checked the `ocis_full/.env` file to see what is default enabled.

Reply:
Our config is different

Reply:
I would keep it as it is.